engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala

```diff
@@ -5,28 +5,34 @@ import pl.touk.nussknacker.ui.security.api.Permission.Permission
 
 sealed trait LoggedUser {
   val id: String
-  def can(category: String, permission: Permission): Boolean
+  val username: Option[String]
   val isAdmin: Boolean
+  def can(category: String, permission: Permission): Boolean
+  def getName: String = username.getOrElse(id)
```

Comment:
It's a trick to save something in not null db column. I would move it closer to the database not to be reused.

Reply:
Indeed @lciolecki can we have `username` as a field as it was before and fill it with the same value in situation when username is the same as id?

Reply:
Ofc I can :)